src/libraries/System.Private.Xml/src/System/Xml/Schema/XmlValueConverter.cs

```diff
@@ -432,6 +445,11 @@ protected string XmlTypeName
             get { return _clrTypeDefault; }
         }
 
+        protected Type? DefaultClrListType
```

Comment:
It would be good to give a quick one line description of what this property is. Similar to the properties above it.